src/PJ_eqearth.c

```diff
@@ -9,14 +9,16 @@ projection, International Journal of Geographical Information Science,
 DOI: 10.1080/13658816.2018.1504949
 
 Port to PROJ by Juernjakob Dugge, 16 August 2018
+Added ellipsoidal equations by Bojan Savric, 22 August 2018
 */
 #define PJ_LIB__
 
+#include <errno.h>
 #include <math.h>
 
 #include "projects.h"
 
-PROJ_HEAD(eqearth, "Equal Earth") "\n\tPCyl., Sph.";
+PROJ_HEAD(eqearth, "Equal Earth") "\n\tPCyl., Sph&Ell\n\th=";
```

Comment:
What's up with the `\n\th=` at the end here? As far as I can tell there's no `+h` parameter present in this projection. Bad cut & paste job?

Reply:
Yes, bad cut & paste.